test/resources/Forward.spec.js

```diff
@@ -58,7 +58,6 @@ function checkForwardResponse ({ t, stelace, method, data }) {
       t.true(headers.authorization.startsWith('Basic'))
 
       if (data) t.is(request.config.data, JSON.stringify(data))
-      else t.is(request.config.data, data)
```

Comment:
Thank you but this is not what I was referring to. To be more explicit: all `checkForwardResponse` calls either have non-empty `data` or `data: undefined`, whereas some call could simply omit `data` as in `checkForwardResponse({ t, stelace, method: 'get' })` to ensure appropriate behavior in such case.

Reply:
Thanks, I misunderstood what you proposed.